### PR TITLE
man: document how to get the boot menu with zero time-out

### PR DESCRIPTION
**Pull request by poettering**
Fixes: #15125

### CODE REVIEW

File: man/systemd-boot.xml [hunk 1]
```diff
@@ -214,6 +214,16 @@
       </varlistentry>
     </variablelist>
 
+    <para>The boot menu is shown when a non-zero menu time-out has been configured. If the menu time-out has
+    been set to zero, it is sufficient to press any key before the boot loader initializes to bring up the
+    boot menu, except for those listed immediately above as they directly boot into the selected boot menu
+    item. Note that depending on the firmware implementation the time window where key presses are accepted
+    before the boot loader initializes might be short. If the window is missed, reboot and try again,
+    possibly pressing a suitable key (e.g. the space bar) continously; on most systems it should be possible
```
Comment (keszybz) [inline man/systemd-boot.xml]: continously → continuously

File: man/systemd-boot.xml [hunk 2]
```diff
@@ -214,6 +214,16 @@
...
+    to hit the time window after a few attempts. To avoid this problem, consider setting a non-zero time-out,
+    thus showing the boot menu unconditionally. Some desktop environments might offer an option to directly
+    boot into the boot menu, to avoid the problem altogether.</para>
```
Comment (keszybz) [inline man/systemd-boot.xml]: Maybe advertise `systemctl reboot --boot-loader-menu` here?

File: man/systemd-boot.xml [hunk 3]
```diff
@@ -214,6 +214,16 @@
       </varlistentry>
     </variablelist>
 
+    <para>The boot menu is shown when a non-zero menu time-out has been configured. If the menu time-out has
```
Comment (keszybz) [inline man/systemd-boot.xml]: This should be "timeout", not "time-out". We predominantly use the first spelling, and the dictionary agrees:

timeout 

     A period of time after which an error condition is raised if
     some event has not occured. ...

time-out

      a brief suspension of play;

File: man/systemd-boot.xml [hunk 4]
```diff
@@ -214,6 +214,17 @@
       </varlistentry>
     </variablelist>
 
+    <para>The boot menu is shown when a non-zero menu timeout has been configured. If the menu timeout has
+    been set to zero, it is sufficient to press any key — before the boot loader initializes — to bring up
```
Comment (bam80) [inline man/systemd-boot.xml:218]: It's still not quite clear - should the user press and hold the key, or just pressing it once is enough? Or repeatedly, to hit the timeframe?

Comment (keszybz) [reply man/systemd-boot.xml:218]: → https://github.com/systemd/systemd/pull/38389

**Pull request merged**